person.rb

```diff
@@ -0,0 +1,39 @@
+require './nameable'
+
+class Person < Nameable
+  attr_accessor :age, :name
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
+
+  def correct_name
+    name
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+end
+
+person = Person.new(22, 'maximilianus')
+person.correct_name
+capitalized_person = CapitalizeDecorator.new(person)
+capitalized_person.correct_name
+capitalized_trimmed_person = TrimmerDecorator.new(capitalizedPerson)
```

Comment:
- [ ] I think you have a typo here that crashed the app when I tried to test it out. IThe correct variable name should be `capitalized_person`.

See error screenshot:
![a1](https://user-images.githubusercontent.com/39523312/157349872-e7d2dfda-d98c-4faf-a5c7-fce8175b735a.png)

Reply:
Yes, sorry. I changed to snake_case and I forgot to change that variable.